packages/@coorpacademy-components/src/molecule/brand-form-group/index.js

```diff
@@ -47,6 +48,8 @@ const buildInput = field => {
       return <ImageUpload {...field} />;
     case 'slider':
       return <SetupSlider {...field} />;
+    case 'sliderCohort':
+      return <SetupSliderCohort {...field} />;
```

Comment:
please remove this, and use `SetupSlider`
your props should look like described at the end of https://github.com/CoorpAcademy/components/pull/1392#issuecomment-442020412

Reply:
@sutrixtuanvothanh ⬆️

Reply:
@chrisdugne , 
so I'll remove `SetupCohortItem` and cutomize `SetupSlide` with my props?

Reply:
It's Done